commands/log_writer_test.go

```diff
@@ -0,0 +1,53 @@
+package commands_test
+
+import (
```

Comment:
Not sure why log_writer is under commands? It almost seems like a helper (just like formcontent), and maybe it should be a separate package? I think the only things under commands should be actual commands that are available to the user to run.